example/Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby File.read(File.join(__dir__, '.ruby-version')).strip
+ruby '>= 2.6.10'
```

Comment:
Why is there a change here?

Reply:
https://react-native-community.github.io/upgrade-helper/?from=0.71.3&to=0.71.7